winpkg/chocolatey/ddev.nuspec

```diff
@@ -20,9 +20,15 @@
     <summary>ddev is a local web development tool optimized for PHP projects and CMSs.</summary>
     <description>ddev allows developers to use both nginx and apache, and many versions of PHP, with no host configurations. It works for most web apps, but has a bias toward TYPO3 and Drupal and Wordpress.</description>
     <releaseNotes>See https://github.com/drud/ddev/releases/tag/vREPLACE_DDEV_VERSION</releaseNotes>
+  <dependencies>
```

Comment:
If you install this 3 tools by choco dependencies and later two of them are installed by the installer too we could finally end in a installation of different versions of this tools. If doing so we should prevent the installation of them in the silent mode which perhaps is not a bad idea at all. I will come up with an PR for the Windows installer...

Reply:
Complicated! But it does seem that if they're using choco, then the choco dependency should be preferred. Of course Chocolatey discourages bundling things as opposed to making them dependencies, but I don't think we have any choice since we certainly don't expect that everybody uses choco.

Reply:
What about adding sudo too here?

Reply:
I looked at it and am open to it. But it looks to me like Chocolatey's sudo is unrelated to the one we've always used.  https://chocolatey.org/packages/Sudo - I wasn't quite able to track its parentage.

Reply:
I say go without it. We always can add it later on...